x/concentrated-liquidity/types/gov.go

```diff
@@ -15,7 +15,7 @@ const (
 
 func init() {
 	govtypes.RegisterProposalType(ProposalTypeCreateConcentratedLiquidityPool)
-	govtypes.RegisterProposalTypeCodec(&CreateConcentratedLiquidityPoolProposal{}, "osmosis/CreateConcentratedLiquidityPoolProposal")
+	govtypes.RegisterProposalTypeCodec(&CreateConcentratedLiquidityPoolProposal{}, "osmosis/CreateCLPoolProposal")
```

Comment:
Why? Is there a limit as well?

Reply:
afaik, yes the same one I have linked previously